packages/fluentui/react-northstar/src/components/Datepicker/Datepicker.tsx

```diff
@@ -0,0 +1,269 @@
+import { Accessibility, datepickerBehavior, DatepickerBehaviorProps } from '@fluentui/accessibility';
+import {
+  getElementType,
+  useAccessibility,
+  useStyles,
+  useTelemetry,
+  useUnhandledProps,
+  ComponentWithAs,
+} from '@fluentui/react-bindings';
+import { Ref } from '@fluentui/react-component-ref';
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import * as _ from 'lodash';
+
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import { FluentComponentStaticProps, ProviderContextPrepared, ShorthandRenderFunction } from '../../types';
+import { commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+import Input from '../Input/Input';
+import Button from '../Button/Button';
+import { CalendarIcon } from '@fluentui/react-icons-northstar';
+import Popup from '../Popup/Popup';
+import {
+  IRestrictedDatesOptions,
+  IDay,
+  DayOfWeek,
+  FirstWeekOfYear,
+  DateRangeType,
+  IDateGridStrings,
+  formatMonthDayYear,
+  IDayGridOptions,
+} from '@fluentui/date-time-utilities';
+import { default as DatepickerCalendar, DatepickerCalendarProps } from './DatepickerCalendar';
+
+const DEFAULT_STRINGS: IDateGridStrings = {
```

Comment:
We already have these defined under `\packages\office-ui-fabric-react\src\components\DatePicker\DatePicker.base.tsx` would it make sense to extract them to the utility package and reuse them?

Reply:
I agree. 

The intent here is that we do not touch the utilities package in this PR and only do a bulk update once we gather more utility feedback and experience.

FYI: @pompomon

Reply:
Yep, we would evaluate if we need the whole localization object to be exported or not in the next steps :)

Reply:
Makes sense, let's add `TODOs` there to ensure that code debt is tracked :)